writing/2016/tacl_colors.tex

```diff
@@ -642,15 +641,15 @@ \subsection{Pragmatic listener}\label{sec:l2}
 
 \subsection{Training} \label{sec:training}
 
-We split our corpus into equal train/dev/test splits (16,635 contexts per split)
+We split our corpus into equal train/dev/test splits (\todocheck{16,635} contexts per split)
 and preprocess the data by (1) lowercasing; (2) tokenizing
 by splitting off punctuation as well as the endings \word{\mbox{-er}}, \word{\mbox{-est}}, and
 \word{\mbox{-ish}};\footnote{We
 only apply this heuristic ending segmentation for the listener; the speaker is trained to produce
 words with these endings unsegmented.} and
 (3) replacing tokens that appear once or not at all
 in the training split with \texttt{<unk>}. We also remove
-listener utterances and concatenate speaker utterances on the same context.
+listener utterances and concatenate speaker utterances on the same context. \todocheck{[rdh: listener utterances are not already removed from the filtered corpus; we could concatenate speaker utterances too?]}.
```

Comment:
that makes sense! I'll add listener utterances back into the corpus. We've always excluded them for the analyses in the R pipeline, though, so nothing to worry about there.

One minor point that just occurred to me: does concatenating utterances make the RNN input a little weird? the sequential statistics of the concatenated utterances are different (e.g. it might learn that a question word usually starting an utterance is likely to appear after a color term usually ending an utterance).